### ISSUE TITLE
Lint json files to prevent syntax errors caused problems.

### ISSUE THREAD

**Issue by PeterDaveHello** [cdnjs/tutorials#25]
Like this:
https://github.com/cdnjs/tutorials/commit/b9c19dad6d291641eab134e477e658f99b3717b8

**Comment by chinmoymohanty85** [comment]
Do you think it is something I can help you with?
If yes, do you have any predefined preference of any linting library?

**Comment by PeterDaveHello** [comment]
@chinmoymohanty85 Yes, every help will be welcome. Actually JSON.parse in nodejs can valid the validation of a josn file.

**Comment by PeterDaveHello** [comment]
@chinmoymohanty85 would you still be interested in this part? Thanks.
